pkg/config/config.go

```diff
@@ -592,6 +593,13 @@ func (c *baseConfig) resolve(builder pathBuilder, fsys fs.FS) error {
 		} else if !filepath.IsAbs(function.ImportMap) {
 			function.ImportMap = filepath.Join(builder.SupabaseDirPath, function.ImportMap)
 		}
+		if len(function.StaticFiles) != 0 {
+			s := []string{}
+			for _, val := range function.StaticFiles {
+				s = append(s, filepath.Join(builder.SupabaseDirPath, val))
+			}
+			function.StaticFiles = s
+		}
```

Comment:
Might need to refer to loadSeedFiles for handling glob patterns.

Also need to handle nested directory when mounting to docker, ie. user may add both html/dir and html/dir/file.html, which docker will reject.

Reply:
> Might need to refer to loadSeedFiles for handling glob patterns.

Edge Runtime already handles the glob patterns.

>Also need to handle nested directory when mounting to docker, ie. user may add both html/dir and html/dir/file.html, which docker will reject.

Config only passes glob patterns of paths to look for static files. The functions directory (and its nested directories) is mounted as before. Edge Runtime would look for matching files based on the patterns in the mounted functions directory. 

Also, users can't add a directory (`html/dir`) as a static file. Edge Runtime will throw an error in both `serve` and `deploy`. It has to be either `html/dir/*.html` or `html/dir/file.html`. Will document them in config docs.

Reply:
Ic, can we also support relative path `../../index.html` like import map? I think it's only a matter of time before user asks for it

Reply:
Only supports paths inside the functions directory. I think it's fine for now, since we want to encourage to keep dependencies of a function inside the same directory.

Reply:
Yes, it's fine as long as we document the intended use case and limitations.

Shall we update the config.toml example too https://github.com/supabase/cli/blob/d27db81d36e480a61cb3738981718740b77f6534/pkg/config/templates/config.toml#L259

Reply:
added in https://github.com/supabase/cli/pull/3045/commits/29a24a2657880ef8299383c03efac59d4a2cd39f